Makefile

```diff
@@ -147,7 +146,12 @@ docs-%:
 ######################################################
 
 .PHONY: travis-all
-travis-all: deps build-linux build-darwin build-windows test perf check
+travis-all:
```

Comment:
Since the build runs inside Docker the wasm targets are getting skipped (because they require docker themselves). The wasm targets are:

```
wasm-build - compiles code inside the wasm directory into the `opa.wasm` module
opa-wasm-test - runs unit tests against the `opa.wasm` module
wasm-test - runs integration tests under test/wasm/ (these compile rego into wasm and execute the wasm inside nodejs)
```

I think the easiest solution would be:

```
travis-build: wasm-build opa-wasm-test
   # current travis-all

travis-all: travis-build wasm-test
```